p4sdk/images/Dockerfile

```diff
@@ -1,27 +1,28 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2024 Intel Corporation
-FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as builder
-ARG TARGETOS
-ARG TARGETARCH
+
+FROM registry.access.redhat.com/ubi9/ubi:9.3-1610
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --no-cache --virtual tar
 COPY . /src
 WORKDIR /src
-RUN mkdir -pv /opt/p4 && tar -zxvf p4.tar.gz -C /opt/p4 && rm -f p4.tar.gz
+RUN mkdir -pv /opt/ && tar -zxvf p4.tar.gz -C /opt/ && rm -f p4.tar.gz
+RUN mkdir -p /usr/share/target_sys/
+RUN cp /opt/p4/p4sde/share/target_sys/zlog-cfg /usr/share/target_sys/zlog-cfg
```

Comment:
If these files are already in there then maybe we do these copy command in the entrypoint.sh script?
Each RUN command adds new layer in the container

Reply:
Done